### PR TITLE
JSON-LD

### PR DESCRIPTION
**Pull request by jphnoel**
This follows #545 and fixes the last comments made by @noirbizarre.

### CODE REVIEW

File: udata/core/user/models.py [hunk 1]
```diff
@@ -196,9 +196,12 @@ def json_ld(self):
             '@type': 'Person',
             '@context': 'http://schema.org',
             'name': self.fullname,
-            'description': mdstrip(self.about),
         }
 
+        description = mdstrip(self.about)
+        if description != '':
```
Comment (davidbgk) [inline udata/core/user/models.py]: Is there a case where `if description` doesn't work?

Comment (jphnoel) [inline udata/core/user/models.py]: If self.about is None, mdstrip returns an empty string. I realized that when re-writing a unit test.

Comment (davidbgk) [inline udata/core/user/models.py]: But `bool('')` returns `False` so is the current comparison needed?

Comment (jphnoel) [inline udata/core/user/models.py]: No, indeed ! I am going to fix this.

File: udata/core/organization/models.py [hunk 2]
```diff
@@ -229,15 +229,26 @@ def check_availability(self):
...
-            'logo': self.logo(external=True),
             'url': url_for('organizations.show', org=self, _external=True),
             'name': self.name,
-            'description': mdstrip(self.description),
         }
 
+        if self.description:
+            result['description'] = mdstrip(self.description)
+
+        logo = self.logo(external=True)
```
Comment (noirbizarre) [inline udata/core/organization/models.py]: I think, self.logo is directly bool-testable aka.:
```
if self.logo:
    result['logo'] = self.logo(external=True)
```

**Pull request merged**